Tasks/VsTestV2/distributedtest.ts

```diff
@@ -99,9 +99,15 @@ export class DistributedTest {
             if (this.inputDataContract.TestSelectionSettings.TestSelectionType.toLowerCase() !== 'testassemblies') {
                 sourceFilter = ['**\\*', '!**\\obj\\*'];
             }
-
+            const telemetryProps: { [key: string]: any; } = { MiniMatchLines: sourceFilter.length };
+            telemetryProps.ExecutionFlow = 'Distributed';
+            var start = new Date().getTime();
             const sources = tl.findMatch(this.inputDataContract.TestSelectionSettings.SearchFolder, sourceFilter);
             tl.debug('tl match count :' + sources.length);
```

Comment:
i know this is not your change but can you change the line to:

tl.debug(`${sources.length} files matched the given minimatch filter`);